app/src/main/java/com/tari/android/wallet/ui/fragment/splash/SplashActivity.kt

```diff
@@ -97,6 +100,19 @@ class SplashActivity : AppCompatActivity() {
         launch(if (exists) AuthActivity::class.java else OnboardingFlowActivity::class.java)
     }
 
+    private fun changeNetwork() {
+        networkRepository.setDefaultNetworkAsCurrent()
+
+        EventBus.walletState.subscribe(this) {
```

Comment:
I was thought that we want to go away from the EventBus.

Reply:
@browmcoat that's true, but it's very difficult to remove it in one place without removing in other. 
I didn't create a new event, just reused existing. 
I'm removing Event Bus event by event. But it always requires a bit of global refactoring. I'll do this while implementing some tasks using event bus.